library/gen_x12_837.inc.php

```diff
@@ -89,35 +89,9 @@ function gen_x12_837($pid, $encounter, &$log, $encounter_claim = false)
     ++$edicount;
 
     if ($claim->federalIdType() == "SY") { // check entity type for NM*102 1 == person, 2 == non-person entity
-        $tempName = $claim->billingFacilityName();
```

Comment:
that would be a separate project, but let me take a look at that class to see how easy it will be to convert to psr4 (if did this, would then give it namespace OpenEMR/Billing which would require appropriate use command in scripts that use it(this actually isn't tough to do with phpstorm), migrate it to library/billing/src/ and place a psr4 entry for it in composer.json).